# duration_path: 
# benchmark_path: 
engine_file: /../../../models/cnnseg/deploy.caffemodel_FP32.trt
range: 60
min_height: -5.0
max_height: 5.0
intensity_normalized: false
score_threshold: 0.6
objectness_thresh: 0.5
height_thresh: 0.5
min_pts_num: 5
use_all_grids_for_clustering: false
sensor_height: 1.8
